src/svs.js

```diff
@@ -6,37 +6,56 @@ const debug = require('debug')('vsac'); // To turn on DEBUG: $ export DEBUG=vsac
 const { Code, ValueSet } = require('cql-execution');
 const vsacCS = require('./vsac-code-systems');
 
-async function downloadValueSet(apiKey, oid, version, output, vsDB = {}, caching = true) {
+async function downloadValueSet(
+  apiKey,
+  oid,
+  version,
+  output,
+  vsDB = {},
+  caching = true,
+  options = { parseCodeSystem: 'replace' }
+) {
   debug(`Getting ValueSet: ${oid}${version != null ? ` version ${version}` : ''}`);
   const params = new URLSearchParams({ id: oid });
   if (version != null) {
     params.append('version', version);
   }
-  const options = {
+  const requestOptions = {
     headers: {
       Authorization: `Basic ${Buffer.from(`apikey:${apiKey}`).toString('base64')}`
     }
   };
   const response = await fetch(
     `https://vsac.nlm.nih.gov/vsac/svs/RetrieveValueSet?${params}`,
-    options
+    requestOptions
   );
   if (!response.ok) {
     throw new Error(response.status);
   }
   const data = await response.text();
-  parseVSACXML(data, vsDB);
+  parseVSACXML(data, vsDB, options);
   if (caching) {
     const file = path.join(output, `${oid}.xml`);
     await fs.writeFile(file, data);
     return file;
   }
 }
 
+function getVSACodeSystem(codeSystems, system) {
```

Comment:
This should be `getVSACCodeSystem` (the last C in VSAC and the first C in Code are different Cs).